From 9bd08c220ca2ab8177ab8cc985086c57c2b71edb Mon Sep 17 00:00:00 2001
From: Eric Sandeen <sandeen@redhat.com>
Date: Mon, 4 Aug 2014 19:05:03 -0400
Subject: [fs] btrfs: remove stale comment from btrfs_flush_all_pending_stuffs

Message-id: <1407179109-13714-417-git-send-email-sandeen@redhat.com>
Patchwork-id: 87291
O-Subject: [RHEL7.1 PATCH 416/422] btrfs: remove stale comment from btrfs_flush_all_pending_stuffs
Bugzilla: 1071179
RH-Acked-by: Zach Brown <zab@redhat.com>

From: David Sterba <dsterba@suse.cz>

commit 0a4eaea892a479aeebccde65986b27cfb6e33a78
Author: David Sterba <dsterba@suse.cz>
Date:   Fri Jun 20 11:31:44 2014 +0200

    btrfs: remove stale comment from btrfs_flush_all_pending_stuffs

    Commit fcebe4562dec83b3f8d3088d77584727b09130b2 (Btrfs: rework qgroup
    accounting) removed the qgroup accounting after delayed refs.

    Signed-off-by: David Sterba <dsterba@suse.cz>
Signed-off-by: Eric Sandeen <sandeen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/btrfs/transaction.c b/fs/btrfs/transaction.c
index 568a320..b10a7df 100644
--- a/fs/btrfs/transaction.c
+++ b/fs/btrfs/transaction.c
@@ -1617,11 +1617,6 @@ static int btrfs_flush_all_pending_stuffs(struct btrfs_trans_handle *trans,
  int ret;
 
  ret = btrfs_run_delayed_items(trans, root);
- /*
-  * running the delayed items may have added new refs. account
-  * them now so that they hinder processing of more delayed refs
-  * as little as possible.
-  */
  if (ret)
   return ret;
 
-- 
1.7.1